content/applications/sales/point_of_sale/payment_methods/terminals/worldline.rst

```diff
@@ -62,15 +62,16 @@ the terminal automatically restarts.
 Configure the payment method
 ----------------------------
 
-From the Point of Sale application, go to :menuselection:`Configuration --> Settings --> Payment
-terminals` and activate the *Worldline* payment terminal.
+First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
+payment method <../../payment_methods>`. Set the journal as :guilabel:`Bank` and select
+:guilabel:`Worldline` in the :guilabel:`Use a Payment Terminal` field. Then, select your terminal
+device in the :guilabel:`Payment Terminal Device` field.
 
 .. image:: worldline/worldline-payment-terminals.png
-   :align: center
 
-Then, go to :menuselection:`Configuration --> Payment methods` and create a new payment method for
-*Worldline*. Select the payment terminal *Worldline* and your payment terminal device on your
-*Payment Method form*.
+Once the payment method is created, you can select it in your POS settings. To do so, go to the
+:ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment method
```

Comment:
```suggestion
:ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment method
```
Why would you :ref: the "Configuration --> Settings" menu when it's at the top of the page and always accessible ? It takes longer for the customer to click on the ref and read the page than just read a menuselection of 3 levels (PoS --> Configuration --> Settings)